tests/architecture/Test.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<rules xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://raw.githubusercontent.com/carlosas/phpat/master/YAML/src/Test/Test.xsd">
+    <test name="testAssertionsImplementAssertionInterface">
```

Comment:
I just realised this tag is called _test_.
The phpat language says that a test contains one or more rules. This should be _rule_ instead of _test_

Reply:
Changed!